syspage.c

```diff
@@ -201,15 +220,15 @@ void syspage_init(void)
 
 /* Syspage's location functions */
 
-int syspage_setAddress(void *addr)
+void syspage_setAddress(void *addr)
 {
-	u32 sz;
+	size_t sz;
 
 	/* Assign syspage to the map */
 	syspage_common.syspage = (void *)addr;
-	sz = (u32)(MAX_SYSPAGE_SIZE / 0x200) * 0x200 + (MAX_SYSPAGE_SIZE % 0x200 ? 0x200 : 0); /* allign to PAGE_SIZE */
+	sz = (size_t)(MAX_SYSPAGE_SIZE / 0x200) * 0x200 + (MAX_SYSPAGE_SIZE % 0x200 ? 0x200 : 0); /* allign to PAGE_SIZE */
```

Comment:
If aligend to `PAGE_SIZE`, `PAGE_SIZE` is different depending on the target:
```
hal/armv7a/zynq7000/config.h:38:#define PAGE_SIZE                0x1000
hal/armv7m/imxrt117x/config.h:40:#define PAGE_SIZE                0x200
hal/armv7m/imxrt106x/config.h:40:#define PAGE_SIZE                0x200
```

Reply:
Yes, it shouldn't be hardcoded. Thanks for paying attention to it.
Fixed.